simulator/infra/ports/module.h

```diff
@@ -9,9 +9,13 @@
  
 #include <infra/log.h>
 #include <infra/ports/ports.h>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
```

Comment:
Could you please use pointers in header and add includes to .cpp file so we can compile Boost only once?

Reply:
It should also resolve name conflicts Visual Studio falsely detects.